server/webapp/WEB-INF/rails.new/app/assets/new_javascripts/views/template_configs/template_row_widget.js.msx

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright 2016 ThoughtWorks, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+define(['mithril', 'helpers/form_helper', 'lodash', 'helpers/mithril_component_mixins', 'js-routes'
+       ], function(m, f, _, ComponentMixins, Routes) {
+  var TemplateRowWidget = {
+    controller: function(args) {
+      var self = this;
+      ComponentMixins.HasViewModel.call(this);
+
+      var vmStateKey = 'show-' + args.template.name();
+      this.vmState(vmStateKey, m.prop(false));
+
+      this.showTemplate = function() {
+        m.route('/' + args.template.name());
+      };
+
+      this.toggleHide = function() {
+        self.vmState(vmStateKey)(!this.vmState(vmStateKey)());
+      };
+
+      this.showState = function() {
+        return self.vmState(vmStateKey)() ? 'show' : 'hide';
+      };
+    },
+
+    view: function(ctrl, args) {
+      var isDeleteAllowed = (args.isUserAdmin && args.template.pipelines().length == 0) ? '' : 'disabled';
+
+      var pipelines = function() {
+        var pipelinesList;
+        if(args.template.pipelines().length !== 0) {
+          pipelinesList = (
+            <dl class="key-value-pair">
+              {_.map(args.template.pipelines(), function(pipeline) {
+                return [
+                  (<dt>
+                    <a href={Routes.editAdminPipelineConfigPath({pipeline_name: pipeline.name()})}>{pipeline.name()}</a>
+                  </dt>),
+                ];
+              })}
+            </dl>
```

Comment:
Can you please check with @naveenbhaskar if using ```<dt>``` alone without ```<dd>``` is a good way to write html. Generally ```<dl>``` holds both ```<dt>``` and ```<dd>```.
Does it need to be a class with "key-value-pair" because I see html doesn't represent any key value pair.